contracts/badger-bridge/BadgerBridgeAdapter.sol

```diff
@@ -125,7 +135,8 @@ contract BadgerBridgeAdapter is OwnableUpgradeable, ReentrancyGuardUpgradeable {
         // darknode args
         uint256 _amount,
         bytes32 _nHash,
-        bytes calldata _sig
+        bytes calldata _sig,
+        bool _mintIbbtc
```

Comment:
Think this needs to go before the darknode args. IIRC the DN args are appended to any user supplied arguments.

For example, here are the arguments we currently specify in the UI:
https://github.com/Badger-Finance/v2-ui/blob/145370eef79e59be49638f80792547976c08a0e2/src/components/Bridge/BridgeForm.tsx#L272-L294